docs/source/tutorials.rst

```diff
@@ -1,82 +1,55 @@
 
-Let's convert a first simple robotic JANI model. An example can be found in `here <https://github.com/convince-project/as2fm/blob/main/jani_generator/test/_test_data/convince_jani/first-model-mc-version.jani>`_. The environment model describes a room with three straight edges and one edge with a small corner in the middle. The room describing the environment in which the robot operates looks like this:
+The following image gives an overview of the complete system:
 
-.. image:: graphics/room.PNG
-    :width: 200
-    :alt: An image illustrating the room's shape.
-
-The lower left corner is at coordinates (0,0) and the upper right corner at coordinates (3,5). Lengths are given in meters, which means that the room has a dimension of 3x5 m with a corner of 0.5m at the top left. 
-The robot is placed at coordinates (0.5, 0.5) initially, and has a round shape with a radius of 0.3 m and a height of 0.2 m. In the small and simple example there are no further obstacles and the robot drives with a linear and angular velocity of 0.5 m/s and 0.5 rad/s, respectively.
-
-The behavior describing how the robot drives around in the room is modeled as a Deterministic Markov Chain (DTMC) shown in the picture below. In each step, the robot moves forward in 50% of the cases and rotates in 50% of the cases. In case it bumps into a wall, it just stops at the collision point and continues operating from there. What is omitted in the picture is the calculation of this collision point and the conversion to and from floats to integers. The latter is only necessary to make the example run in Storm because the tool currently does not support transient floats.
-
-.. image:: graphics/dtmc.PNG
+.. image:: graphics/scxml_tutorial_ros_example_w_bt.drawio.svg
     :width: 800
-    :alt: An image of the DTMC representing the robot's behavior.
-
-The property given in the JANI file checks for the minimal probability that eventually within 10 000 steps the position (1.0, 1.0) is reached with an error range of 0.05 m.
+    :alt: An image of the complete exemplary system.
 
+In this example, the system is composed by the following components modeled in SCXML:
 
+* a **battery_drainer**, that at each time step drains the battery by 1%, and each time the charge trigger is received, it recharges the battery to 100%.
+* a **battery_manager**, that at each time the battery level is received checks if it is below 30% and, if so, triggers the alarm.
 
+The **behavior tree** continuously checks the alarm topic and, once it is triggered, sends a charge trigger to the battery_drainer.
 
+The JANI property given in `battery_depleted.jani <https://github.com/convince-project/as2fm/tree/main/jani_generator/test/_test_data/ros_example/battery_depleted.jani>`_ defines the property of interest to be model checked. In this case, it calculates the minimal probability that the battery level is below or equal to zero eventually, i.e., all we verify here is that the battery is empty at some point.
```

Comment:
Given that we also provide a bt, it would make more sense to use the version where the battery is also charged and verify that it never runs out. I think this would make a bit more sense. Otherwise, people could get hung up on the point that the bt is useless.

Reply:
Right, but we do not have that property implemented yet. We only have the emtpy, belowe 20% and alarm on.
I can implement one more property for the battery to get charged after a minimum amount of time